odom/odom/config/localization.yaml

```diff
@@ -0,0 +1,52 @@
+frequency: 50
+
+# Specify planar environment
+two_d_mode: true
+
+map_frame: map
+odom_frame: caffeine/odom
+base_link_frame: caffeine/base_link
+world_frame: caffeine/odom
+
+# publish transform: world_frame --> base_link_frame
+publish_tf: true
+
+transform_time_offset: 0.05
+
+# See /diagnostics topic for debug
+print_diagnostics: true
+
+## Odometry Sensor(s) ##
+
+odom0: caffeine/odom
+odom0_config: [true,  true,  false,
+               false, false, true,
+               true,  true,  false,
+               false, false, true,
+               false, false, false]
+odom0_queue_size: 10
+odom0_differential: false 
+odom0_relative: false
+
+odom1: zed/zed_node/odom
+odom1_config: [false, false, false,
+               false, false, false,
+               true,  true,  false,
+               false, false, true,
+               false, false, false]
+odom1_queue_size: 10
+odom1_differential: true
+odom1_relative: false
+
+## IMU Sensor(s) ## 
+
+imu0: caffeine/imu/data 
+imu0_config: [false, false, false, 
+              true , true , true ,
+              false, false, false,
+              true , true, true, 
+              true , true, true]
```

Comment:
Since we set `two_d_mode: true` we should set any measurements that pertain to "z", "roll" and "pitch" to `false`.